frontend/.storybook/main.js

```diff
@@ -11,4 +13,11 @@ module.exports = {
       },
     },
   ],
+  webpackFinal: async config => {
+    [].push.apply(config.resolve.plugins, [
```

Comment:
Just curious as to why you needed to use apply() here?

Reply:
uh i guess this could be a map as well, honestly i don't know since this is copypasta - @thaigillespie do you think we need to change anything? this is just a config in .storybook

Reply:
Nah no need to change anything, I'm just a bit confused by that line